packages/components/src/Modal/docs/custom-trigger.tsx

```diff
@@ -7,7 +7,7 @@ export default function Example() {
     return (
         <>
             <Button variant="secondary" onPress={() => setIsOpen(true)}>Open modal</Button>
-            <Modal isOpen={isOpen} onOpenChange={setIsOpen}>
+            <Modal overlayProps={{ isOpen, onOpenChange: setIsOpen }}>
```

Comment:
Oufff, i tought we were duplicating the props on the overlay and modal, can't we keep <Modal isOpen={isOpen} onOpenChange={setIsOpen}>?